app/src/components/market/common/list_item/index.tsx

```diff
@@ -138,8 +138,11 @@ export const ListItem: React.FC<Props> = (props: Props) => {
 
   let currentPrediction
   let unit
+  let scalarTitle
+
   if (isScalar) {
     unit = title.split('[')[1].split(']')[0]
+    scalarTitle = title.split(' [')[0]
```

Comment:
We noticed in a previous PR that retrieving/removing the unit like that can cause a bug if there are square brackets used in the title at all as it will grab the wrong thing. To fix this we have `getUnit()`, which I see is actually unused here. Can you use `getUnit` for the unit here and also create and use a similar function to `getUnit` but for removing the unit from the title?

Reply:
Good suggestion! Will use it